modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/util/RelayUtils.java

```diff
@@ -149,8 +149,8 @@ public static void builldMessage(MessageContext messageContext, boolean earlyBui
             try {
                 bufferedInputStream.reset();
                 bufferedInputStream.mark(0);
-            } catch (IOException e) {
-                handleException("Error while checking bufferedInputStream", e);
+            } catch (Exception e) {
+                // just ignore the error
```

Comment:
Why are we ignoring the exception here? Won't this make debugging issues harder?

Reply:
@malinthaprasan Shall we fix it using a separate commit eventhough this comes from the revert?

Reply:
+1. Let's do it as a different PR. But I guess we have to merge it as it introduces a regression.